LICENSE-MIT

```diff
@@ -1,5 +1,3 @@
-Copyright (c) 2016 The Miri Developers
```

Comment:
Doesn't removing this conflict with "The above copyright notice... shall be included in all copies or substantial portions of the Software"? Although it seems that this notice was removed from rustc

Reply:
This was changed in rustc in https://github.com/rust-lang/rust/pull/43498 without explicit buy-in from all contributors.

So this particular change I think we are safe to do in Miri as well.